Corona-Warn-App/src/main/res/layout/fragment_information_privacy.xml

```diff
@@ -25,7 +26,9 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:focusable="true"
-            app:title="@string/information_privacy_title" />
+            app:title="@string/information_privacy_title"
+            bind:navigationButtonId="@{@id/header_button_back}"
+            bind:titleTextViewId="@{@id/information_privacy_header}" />
```

Comment:
maybe call it `automation:*` so we know it is just for automation  , same for other layouts